addon/components/mapbox-map.js

```diff
@@ -52,4 +52,12 @@ export default Ember.Component.extend({
       this.set('map', map);
     });
   },
+
+  zoomOrCenterChanged: Ember.observer('zoom', 'center', function() {
```

Comment:
Observers are more than bad... They way should be handled is: When you're setting the zoom or center, you set the view. There might be situations where you don't want this behaviour by default and you can't disable an observer

Reply:
All right 👍 Will change the implementation and close this